CHANGELOG.md

```diff
@@ -3,10 +3,17 @@
 FEATURES:
 
 * **New Data Source:** `vcd_org` Organization - ([#218](https://github.com/terraform-providers/terraform-provider-vcd/issues/218))
+* **New Data Source:** `vcd_catalog` Catalog - ([#218](https://github.com/terraform-providers/terraform-provider-vcd/issues/218))
+* **New Data Source:** `vcd_catalog_item` CatalogItem - ([#218](https://github.com/terraform-providers/terraform-provider-vcd/issues/218))
```

Comment:
Links will be automatically populated upon release. Same with above two lines.

```suggestion
* **New Data Source:** `vcd_catalog_item` CatalogItem - [#218]
```

Reply:
fixed